We prefer amd64/sparcv9 over 64.  It fits better our packaging.

--- openssl-3.4.1/Configurations/10-main.conf.orig
+++ openssl-3.4.1/Configurations/10-main.conf
@@ -271,7 +271,7 @@
         perlasm_scheme   => "elf",
         shared_cflag     => "-fPIC",
         shared_ldflag    => add_before("-shared -static-libgcc"),
-        multilib         => "/64",
+        multilib         => "/amd64",
     },
 
 #### Solaris x86 with Sun C setups
@@ -335,7 +335,7 @@
         inherit_from     => [ "solaris-sparcv9-gcc" ],
         cflags           => sub { my $f=join(" ",@_); $f =~ s/\-m32/-m64/; $f; },
         bn_ops           => "BN_LLONG RC4_CHAR",
-        multilib         => "/64",
+        multilib         => "/sparcv9",
     },
 
 #### SPARC Solaris with Sun C setups
